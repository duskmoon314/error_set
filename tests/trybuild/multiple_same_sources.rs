use error_set::error_set;

error_set! {
    X {
        IoError(std::io::Error),
        IoError2(std::io::Error),
    },
    Y {
        IoError2(std::io::Error),
    },
}

pub fn main() {}
