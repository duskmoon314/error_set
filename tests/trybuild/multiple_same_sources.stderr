error: A variant with source 'std::io::Error' already exists in error enum 'X'
 --> tests/trybuild/multiple_same_sources.rs:6:18
  |
6 |         IoError2(std::io::Error),
  |                  ^^^^^^^^^^^^^^
